plugins/woocommerce/src/Admin/API/Reports/Controller.php

```diff
@@ -151,6 +151,11 @@ protected function get_order_number( $order_id ) {
 
 		if ( 'shop_order_refund' === $order->get_type() ) {
 			$order = wc_get_order( $order->get_parent_id() );
+
+			// If the parent order doesn't exist, return null.
+			if ( ! $order instanceof \WC_Order && ! $order instanceof \WC_Order_Refund ) {
```

Comment:
It might be better to wrap this check into a method as it has been used four times in this code.

Reply:
Thanks done here: [b93e55e](https://github.com/woocommerce/woocommerce/pull/49006/commits/b93e55eef5a9c3f3df88181b79013a2874d58472)